lib/src/main/java/com/auth0/jwt/verification/VerificationAndAssertion.java

```diff
@@ -137,14 +137,14 @@ private static void assertDateIsPast(Date date, long leeway, Date today) {
         }
     }
 
-    private static void assertValidAudienceClaim(List<String> audience, List<String> value) {
-        if (audience == null || !audience.containsAll(value) || audience.size() != value.size()) {
+    private static void assertValidAudienceClaim(List<String> actual, List<String> expected) {
+        if (expected == null || !expected.containsAll(actual)) {
```

Comment:
I think this is wrong. The required audiences (the ones in `expected`) must **all** be contained in `actual` at the bare minimum. If `actual` contains additional audiences it's fine, as long as it contains **all** the `expected` ones. So rollback this line to the previous clause:

```java
actual == null || !actual.containsAll(expected)
```

Reply:
done